ledger/cow.go

```diff
@@ -56,6 +56,7 @@ type roundCowState struct {
 	commitParent *roundCowState
 	proto        config.ConsensusParams
 	mods         ledgercore.StateDelta
+	groupIdx     int
```

Comment:
move down to trackedCreatables and document by saying it is a group index of a current transaction used in conjunction with trackedCreatables

Reply:
I'm not sure I agree.  It's the standard group index, right?  It may only be used with trackedCreatables now, but it has a real meaning that makes sense on its own.

Reply:
cow is kind of transaction-agnostic and builds up own state while processing a transaction. So until it is committed to parent is has data relevant to `groupIdx`. Parent cow is a sum of all deltas, and `groupIdx` does not make sense there (it does not contains data from txn 0 applying). Same for `trackedCreatables`, it only has valid data within a group evaluation.
A clean solution might be `txnGroupCowState` that includes `roundCowState`  (well, naming) but it looks like as overkill here. So grouping data with a limited meaning IMO must.